### PR TITLE
[Load] Set the load channel's timeout to be the same as the load job's timeout

### PR DESCRIPTION
**Pull request by morningman**
When performing a long-time load job, the following errors may occur. Causes the load to fail.

`load channel manager add batch with unknown load id: xxx`

There is a case of this error because Doris opened an unrelated channel during the load
 process. This channel will not receive any data during the entire load process. Therefore,
 after a fixed timeout, the channel will be released.

And after the entire load job is completed, it will try to close all open channels. When it try to
 close this channel, it will find that the channel no longer exists and an error is reported.

This CL will pass the timeout of load job to the load channel, so that the timeout of load channels
will be same as load job's.

ISSUE #2404

### CODE REVIEW

File: gensrc/thrift/DataSinks.thrift [hunk 1]
```diff
@@ -104,6 +104,7 @@ struct TOlapTableSink {
     11: required Descriptors.TOlapTablePartitionParam partition
     12: required Descriptors.TOlapTableLocationParam location
     13: required Descriptors.TPaloNodesInfo nodes_info
+    14: optional load_channel_timeout_s // the timeout of load channels in second
```
Comment (imay) [inline gensrc/thrift/DataSinks.thrift]: forget type?

File: fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java [hunk 2]
```diff
@@ -580,7 +580,6 @@ protected void unprotectedExecuteCancel(FailMsg failMsg, boolean abortTxn) {
             }
         }
         idToTasks.clear();
-        loadStatistic.clearAllLoads();
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java]: is it OK to delete this?

**Review by imay — CHANGES_REQUESTED** [review]

**Comment by chaoyli** [comment]
How to distinguish the unrelated channel and related channel?

File: gensrc/thrift/DataSinks.thrift (hunk 1 above)
Comment (morningman) [reply gensrc/thrift/DataSinks.thrift]: I miss it

File: fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java (hunk 2 above)
Comment (morningman) [reply fe/src/main/java/org/apache/doris/load/loadv2/LoadJob.java]: It OK, there is only some informations for showing.
And I just want to see how many rows it read after being cancelled.

**Comment by morningman** [comment]
> How to distinguish the unrelated channel and related channel?

You misunderstood, "unrelated" means channel without data, not "unrelated to the load job".

**Review by imay — APPROVED** [review]
LGTM

**Pull request merged**